ports/esp32/boards/DENEYAP_KART/modules/deneyapkart.py

```diff
@@ -0,0 +1,34 @@
+# Deneyap Kart Helper Library
+
+from micropython import const
+from machine import Pin
+
+# Pin Assignments
+
+# SPI
+SPI_MOSI = const(5)
+SPI_MISO = const(18)
+SPI_CLK = const(19)
+
+# I2C
+I2C_SDA = const(4)
+I2C_SCL = const(15)
+
+# DAC
+DAC1 = const(25)
+DAC2 = const(26)
+
+# RGB LED
+LEDR = const(3)
+LEDG = const(1)
+LEDB = const(4)
+
+# BUTTON
+GPKEY = const(0)
```

Comment:
There's no problem with configuring these pins this way but, just to let you know, #12010 added support for _named pins_. The idea is to use a `pins.csv` file to specify an _alias_ to a pin. So you could alias `SPI_MOSI` to `Pin(5)` and access it with `machine.Pin.board.SPI_MOSI` or `machine.Pin(5)`.

See [ARDUINO_NANO_ESP32](https://github.com/micropython/micropython/tree/master/ports/esp32/boards/ARDUINO_NANO_ESP32) for an example.